packages/nocodb/src/lib/utils/projectAcl.ts

```diff
@@ -26,6 +26,7 @@ export default {
     dataGroupBy: true,
     commentsCount: true,
     exportCsv: true,
+    exportCsvData: true,
```

Comment:
What's difference with between `exportCsv` and `exportCsvData`? Why not using `exportExcel`?

Reply:
The xlsx file is created on the frontend. The backend send data as CSV (just data, not file) and the frontend use this data to build a XLSX file.
That's why the endpoint I created is not `excel` or `xlsx` because the backend doesn't know about all theses things, it only sends CSV file or CSV data.